### PR TITLE
Added a new Python script and some changes in existing one

### PR DESCRIPTION
**Pull request by SKAUL05**
Python Program to send a message to Slack Channel.

### CODE REVIEW

File: web_programming/slack_message.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created by sarathkaul on 12/11/19
+
+import json
+import requests
+
+
+def send_slack_message(message_body, slack_url):
+    slack_data = {"text": message_body}
+    headers = {"Content-Type": "application/json"}
+    response = requests.post(slack_url, data=json.dumps(slack_data), headers=headers)
```
Comment (cclauss) [inline web_programming/slack_message.py]:
```python
   response = requests.post(slack_url, json={"text": message_body}, headers=headers)
```
And remove lines 3 and 8.

File: web_programming/slack_message.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    if response.status_code != 200:
+        raise ValueError(
+            "Request to slack returned an error %s, the response is:\n%s"
+            % (response.status_code, response.text)
+        )
```
Comment (cclauss) [inline web_programming/slack_message.py]:
```python
        raise ValueError(f"Request to slack returned an error {response.status_code}, "
                         f"the response is:\n{response.text}")

File: web_programming/slack_message.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Created by sarathkaul on 12/11/19
+
+import json
+import requests
+
+
+def send_slack_message(message_body, slack_url):
```
Comment (cclauss) [inline web_programming/slack_message.py]:
```python
def send_slack_message(message_body: str, slack_url: str) -> None:
```

**Comment by SKAUL05** [comment]
@cclauss Updated the changes requested

**Pull request merged**